Hussin 02 html setup

HussinSaheb opened the pull request.

HussinSaheb committed:
added tags for inital design setup

HussinSaheb committed:
updated readme

HussinSaheb committed:
added css file for some initial design

HussinSaheb committed:
added jasmine and spec file to test functions

HussinSaheb committed:
wrote functions for display the guess, and display the player names

AEJackson commented on index.html:
```diff
@@ -1,10 +1,43 @@
+          </div>
+        </section>
+        <section class="game">
+        <div class="row">
```
check your indentation here

AEJackson commented on js/index.js:
```diff
@@ -1,6 +1,51 @@
+    var personName = "";
+    // create player div
+    for (var i = 0; i < num; i++) {
+      personName= arrayNames[i];
```
Is "personName" being used ?

HussinSaheb committed:
Fixed indentation

HussinSaheb committed:
personName explained

AddPlayers adds players name that is retrieved from another function to the main game window.

HussinSaheb committed:
added comments

HussinSaheb committed:
removed unused variables
